### ISSUE TITLE
kali-rolling.sh - Package 'php5' has no installation candidate

### ISSUE THREAD

**Issue by ggrochow** [g0tmi1k/os-scripts#22]
php5 was removed from the kali-rolling sources

http://pkg.kali.org/pkg/php5
>  [2016-07-12] php5 5.6.22+dfsg-2 removed from kali-rolling (Kali Repository) 

Breaks on these steps
```
[+] (50/188) Installing rips ~ source code scanner
    E: Package 'php5' has no installation candidate

[+] (97/188) Installing b374k ~ (PHP) web shell
    E: Package 'php5-cli' has no installation candidate

[+] (170/188) Installing apache2 & php5 ~ web server
    E: Package 'php5' has no installation candidate
    E: Package 'php5-cli' has no installation candidate
    E: Package 'php5-curl' has no installation candidate

```

**Comment by sheimo** [comment]
Gotta add the old repository to your source list. 

echo 'deb http://old.kali.org/kali sana main non-free contrib' >> /etc/apt/sources.list && apt-get update

**Comment by g0tmi1k** [comment]
Thanks for the bug report.

This has been fixed in b970909a2a9e15df87ddd3ec5834e0fa01fdc12c.

---

p.s. @sheimo Mixing in the the kali old repo is not a wise/smart thing to-do...

**Comment by ghost** [comment]
> Gotta add the old repository to your source list.
> 
> echo 'deb http://old.kali.org/kali sana main non-free contrib' >> /etc/apt/sources.list && apt-get update

THANKS YOU! Wokrs Perfect for MEE!! (I'm using WSL2 on Windows 10) Great...
